README.md

```diff
@@ -109,10 +123,11 @@ STATICFILES_DIRS = (
 Restart the apache server to put all changes into effect. 
 
 ```
-/etc/init.d/apache2 reload
+sudo /etc/init.d/apache2 reload
 ```
 
-###Migrate the Database Schema using South
```

Comment:
We should put it SOMEWHERE so we don't forget.  But I don't think here is the right place